ts/DataGrid/DataGrid.ts

```diff
@@ -279,9 +340,7 @@ class DataGrid {
     /**
      * Get a reference to the underlying DataTable from options, or create one
      * if needed.
-     *
-     * @return {Highcharts.DataTable}
-     * Table reference
+     * @internal
```

Comment:
Why removing the return description?

Reply:
It is obvious from the description and the type can be reflected.